diff --git a/Source/driver/main.cpp b/Source/driver/main.cpp
index 1b8b023dc..192c3ec85 100644
--- a/Source/driver/main.cpp
+++ b/Source/driver/main.cpp
@@ -2,7 +2,7 @@
 #include <new>
 #include <cstdio>
 #include <cstring>
-#include <format>
+#include <cstdio>
 #include <iostream>
 #include <iomanip>
 
@@ -256,8 +256,9 @@ main (int   argc,
         //
         // It's actually the high water mark of heap space required by FABs.
         //
-        std::cout << std::format("CPU({}): Heap Space (bytes) used by Coalescing FAB Arena: {}\n",
-                                 ParallelDescriptor::MyProc(), arena->heap_space_used());
+        std::cout << "CPU(" << ParallelDescriptor::MyProc()
+                  << "): Heap Space (bytes) used by Coalescing FAB Arena: "
+                  << arena->heap_space_used() << "\n";
     }
 
     BL_PROFILE_VAR_STOP(pmain);
diff --git a/external/amrex b/external/amrex
--- a/external/amrex
+++ b/external/amrex
@@ -1 +1 @@
-Subproject commit 2fd316b7e939ce8e1c0b9d1468c67ab799f34e9d
+Subproject commit 2fd316b7e939ce8e1c0b9d1468c67ab799f34e9d-dirty
